CHANGES.rst

```diff
@@ -9,6 +9,17 @@ Changes
 
 * Nothing yet.
 
+`3.3.1 <https://github.com/Ouranosinc/Magpie/tree/3.3.1>`_ (2020-12-02)
+------------------------------------------------------------------------------------
```

Comment:
revert only the header part to keep 'latest' and remove the tmp version
we will do an actual bump in master to generate the tags/dockers

Reply:
So, that would mean changing the highlighted line to this :
`Unreleased <https://github.com/Ouranosinc/Magpie/tree/master>`_ (latest)
?
I wasn't sure about the good practice here with version bumping. I was wondering if I should have done it after the PR was approved. The problem was I wanted to reference the version in the test file interfaces.py, for the skip safeguards, so I bumped the version to make it clear I was referencing the current version in the skip safeguards.

Reply:
Yes, I understand what you mean. I do the same kind of hack locally for testing new versions. 
The only thing is that since there isn't an actual tag and version, I don't want them to be displayed in the doc pushed to readthedocs when it gets merged. 

I haven't found a clean way to solve this yet.

Also, I just noticed, your compared should use `3.4.0` instead of `3.3.1`.

Reply:
try with this patch: 
https://github.com/Ouranosinc/Magpie/tree/test-version